deepchecks/vision/checks/performance/robustness_report.py

```diff
@@ -271,31 +299,6 @@ def _create_top_affected_graph(self, top_affected_dict):
         return fig
 
 
-def get_robustness_augmentations(dataset):
-    transform_type = dataset.get_transform_type()
-    multi_channel = dataset.data_dimension > 1
-    if transform_type == 'albumentations':
-        # Note that p=1.0 since we want to apply those to entire dataset
-        augmentations = [
-            albumentations.RandomBrightnessContrast(p=1.0),
-            albumentations.ShiftScaleRotate(p=1.0),
-        ]
-        if multi_channel:
```

Comment:
What happens here?

Reply:
There are augmentations that doesn't work on black-white images